libs/core/src/integration/email.schemas.ts

```diff
@@ -101,9 +101,15 @@ export const EnrichedThread = Thread.extend({
   icon_url: z
     .string()
     .describe('The icon url of the community that the thread belongs to'),
+  author: z.string().nullish().optional(),
 });
 
 export const GetDigestEmailData = {
-  input: z.object({}),
-  output: z.record(z.string(), z.array(EnrichedThread)),
+  input: z.object({
+    user_id: z.string(),
+  }),
+  output: z.object({
+    threads: z.array(EnrichedThread),
+    numberOfThreads: z.number(),
```

Comment:
numberOfThreads seems redundant if same as the length of the threads array

Reply:
we need this to add condition in knock . i did not find way to check array length there 
<img width="445" alt="image" src="https://github.com/user-attachments/assets/c12a982d-76b0-49e9-9dd0-6f933816d1b4" />